examples/python/cluster_example.py

```diff
@@ -0,0 +1,125 @@
+import asyncio
+from typing import List, Tuple
+
+from glide import (
+    AllNodes,
+    ClosingError,
+    GlideClusterClient,
+    GlideClusterClientConfiguration,
+    InfoSection,
+    Logger,
+    LogLevel,
+    NodeAddress,
+    RequestError,
+)
+
+
+async def create_client(
+    nodes_list: List[Tuple[str, int]] = [("localhost", 47243)]
+) -> GlideClusterClient:
+    """
+    Creates and returns a GlideClusterClient instance.
+
+    This function initializes a GlideClusterClient with the provided list of nodes.
+    In cluster mode, add the address of any node; the client will automatically
```

Comment:
Please rephrase this comment. It is unclear from the comment:

- Does the user need to provide all of the nodes in the cluster or just some?, what about replicas?
- Why do we need to state "In cluster mode", I thought that this is clearly a "cluster" example (in addition, this function creates `GlideClusterClient`)